domains/eliasbloem.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "ItzjustElias",
+        "email": "elias.bloem123@gmail.com"
+    },
+    "record": {
+        "A": [
+            "75.2.60.5"
```

Comment:
Can you show a preview of your website?

Reply:
[](https://6725e1efb9448f0008cbe453--eliasbloem.netlify.app)
or do you need an image